git/README.md

```diff
@@ -0,0 +1,44 @@
+# Git
+
+Git is pretty awesome. Here's some reasonable standards when it comes to using it.
+
+## Commit Messages
+
+At Mobify, we recommend a few standards regarding git commit messages:
+- If the commit is related to a JIRA ticket, prefix the case-sensitive ticket number to the start of the message
```

Comment:
@yourpalsonja Yes.  Each commit that was merged as part of a PR has a link back to the PR on it. 

![screen_shot_2016-08-26_at_1_53_59_pm](https://cloud.githubusercontent.com/assets/77138/18020137/b0241092-6b94-11e6-81d5-500c72918e8a.png)